vassal-app/src/main/java/VASSAL/build/GameModule.java

```diff
@@ -2466,16 +2466,9 @@ private void buildCrc() {
       }
     }
 
-    try {
-      // Base CRC is of module file only
-      crc = CRCUtils.getCRC(List.of(files.get(0)));
-      // CombinedCrc includeds extensions as well
-      combinedCrc = CRCUtils.getCRC(files);
-    }
-    catch (IOException e) {
-      log.error("Error generating CRC", e); //NON-NLS
-      return;
-    }
+    crc = CRCUtils.getCRC(List.of(files.get(0)));
```

Comment:
Not sure if it is possible, but just to be paranoid, could check that `files` contains at least one file (i.e. not empty).

Otherwise, if it is ok to throw an `IndexOutOfBoundsException`, than that should be added to the documentation of this function. If it throws, `combinedCrc` won't be initialized.

Reply:
No, the first file is guaranteed to be the module file.